python/GafferArnoldUITest/LightVisualiserTest.py

```diff
@@ -0,0 +1,107 @@
+##########################################################################
+#
+#  Copyright (c) 2020, Cinesite VFX Ltd. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of Cinesite VFX Ltd. nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import unittest
+
+import IECore
+
+import Gaffer
+import GafferArnold
+import GafferScene
+import GafferSceneUITest
+import GafferTest
+
+@unittest.skipIf( GafferTest.inCI(), "OpenGL not set up" )
+class LightVisualiserTest( GafferSceneUITest.SceneUITestCase ) :
+
+	@GafferTest.TestRunner.PerformanceTestMethod()
+	def testDrawPerformanceQuad( self ) :
+
+		script = Gaffer.ScriptNode()
+
+		script["quad"] = GafferArnold.ArnoldLight()
+		script["quad"].loadShader( "quad_light" )
+
+		instancerOut = self.setupInstancer( script["quad"]["out"] )
+
+		self.benchmarkRender( instancerOut )
+
+	@GafferTest.TestRunner.PerformanceTestMethod()
+	def testDrawPerformanceSpot( self ) :
```

Comment:
The two light tests took a total of 286s here - waay longer than can be tolerated for the standard test suite.